simplified-app-shared/build.gradle

```diff
@@ -113,6 +113,7 @@ dependencies {
   api libraries.readiumSharedJS
   api libraries.slf4j
   api libraries.viewPagerIndicator
+  api libraries.instabug
```

Comment:
I'm OK with this for now, given that there's no API, but we will need to hide the setup behind an interface when we do the repository separation in order to make it into an optional dependency. The reason being: Declaring this in the `app-shared` module means that all app frontends that depend on the app-shared module _must_ now use Instabug unconditionally. LFA in particular won't be able to do this for many deployments.